### PR TITLE
feat(global): add quarterly closure

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

**Comment by velrest** [comment]
# Pending Request
image
image

**Comment by velrest** [comment]
# BFD Action needed
This is mostly the case after a municipality calls `closeQuarterlySurvey`.
image
image

**Comment by velrest** [comment]
# Municipality action required
image
image

**Comment by velrest** [comment]
# Errors need fixing
image
image

**Comment by velrest** [comment]
# Finalized
image
image

**Comment by luytena** [comment]
Regarding the UI: 
In the tiles: I would maybe add the icon and the label of the status on the same line and use the same style as the "Bald verfügbar!" tag.

File: addon/components/quarterly-closure/close-survey-button.hbs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div>
+  <UkButton
+    @color="primary"
+    @loading={{this.closeQuarterlySurvey.isRunning}}
+    @disabled={{this.disabledButton}}
+    {{on "click" this.closeQuarterlySurvey.perform}}
```
Comment (luytena) [inline addon/components/quarterly-closure/close-survey-button.hbs:6]: Use perform helper?

File: addon/components/quarterly-closure/close-survey-button.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  [CLOSED]: "success",
+};
+
+export default class QuarterlyClosureStatus extends Component {
+  @service quarterlyClosure;
+  @service notification;
+  @service intl;
+
+  get disabledButton() {
+    return ![620, 630, 650, 651, 663, 665].includes(
```
Comment (luytena) [inline addon/components/quarterly-closure/close-survey-button.js:24]: Maybe replace with constants

File: addon/helpers/current-quarter.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { helper } from "@ember/component/helper";
+import { DateTime } from "luxon";
+
+const currentQuarter = function isUnset([format = "q/yyyy"]) {
```
Comment (luytena) [inline addon/helpers/current-quarter.js:4]: Should display quarter being worked on, not by date

File: translations/quarterly-closure/de.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      640: Formeller Abschluss wurde nicht erteilt
+      650: Formeller Abschluss wurde nicht erteilt
+      651: Formeller Abschluss wurde nicht erteilt
+      660: Formeller Abschluss durch die Gemeinde gemacht
+      661: Abschluss der Erhebung durch Gemeinde erteilt
+      662: Formeller Abschluss erteilt. Automatische Anpassung nach 7 Tagen oder manuelle Anpassung des Status
+      663: Definitiver Abschluss nicht bestätigt
+      665: Definitiver Abschluss nicht bestätigt
+      667: Abschluss der Erhebung durch Gemeinde
+      670: Kontrollliste erstellen und Qualitätskontrollen durchfühen
```
Comment (luytena) [inline translations/quarterly-closure/de.yaml:24]:
```suggestion
      670: Kontrollliste erstellen und Qualitätskontrollen durchführen
```

**Review by luytena — APPROVED** [review]

**Pull request merged**